.vscode/settings.json

```diff
@@ -0,0 +1,14 @@
+{
+  "python.autoComplete.extraPaths": [
+    "${workspaceFolder}/sources/poky/bitbake/lib",
+    "${workspaceFolder}\\sources\\poky\\meta\\lib"
+  ],
+  "python.analysis.extraPaths": [
+    "${workspaceFolder}/sources/poky/bitbake/lib",
+    "${workspaceFolder}\\sources\\poky\\meta\\lib"
+  ],
+  "files.associations": {
+    "*.conf": "bitbake",
+    "*.inc": "bitbake"
+  }
+}
```

Comment:
I guess you included this file in the commit by accident. Would you please remove it?

Reply:
ugh the bitbake vs code extension generated it and i didn't spot it sorry, ill remove it.